kafo_parsers.gemspec

```diff
@@ -16,13 +16,15 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.required_ruby_version = '>= 2.7', '< 4'
```

Comment:
This is one of those things that I always put in its own commit because it's WAY more important than the other parts. This impacts the usage elsewhere while the other bits less so (though runtime dependencies do matter as well). It's still OK to put in the same PR, but then I'd also change the PR summary to reflect it.

Reply:
@ekohl https://github.com/theforeman/kafo_parsers/pull/52

Thanks for the mention!